lms/static/sass/bootstrap/_footer.scss

```diff
@@ -0,0 +1,65 @@
+// Open edX: LMS footer
+// ====================
+
+.wrapper-footer {
+  box-shadow: 0 -1px 5px 0 rgba(0,0,0,0.1);
+  border-top: 1px solid #c5c6c7;
+  background-color: $white;
+  margin-top: 40px;
+  padding: 20px;
+  font-family: $font-family-sans-serif;
+
+  footer {
+    max-width: 1140px; // Use map-get($container-max-widths, xl) once theming complete
+    margin: 0 auto;
+
+    .site-nav {
+      padding: 0;
+
+      .nav-item {
+        margin-right: 20px;
```

Comment:
This reminds me: we need to add in postcss processing to handle RTL. I've made a story for this:

https://openedx.atlassian.net/browse/LEARNER-1705